openassessment/xblock/grade_mixin.py

```diff
@@ -347,11 +347,10 @@ def _get_assessment_part(title, part_criterion_name, assessment):
                     for index, peer_assessment in enumerate(peer_assessments)
                 ],
             }
-        elif "peer-assessment" in assessment_steps:
-            peer_assessment_part = {
-                'title': _('Peer Median Grade'),
-                'option': {'label': _('Waiting for peer reviews')}
-            }
+            # If we weren't able to get a median option when there should be one, show the following message
```

Comment:
Nit: can this functionality move into _peer_median_option? It already has criterion, and then the contract will be option always returns a label.